source/client/output_transform_main.cc

```diff
@@ -58,7 +58,12 @@ uint32_t OutputTransformMain::run() {
   }
   OutputFormatterFactoryImpl factory;
   OutputFormatterPtr formatter = factory.create(translated_format);
-  std::cout << formatter->formatProto(output);
+  auto format_status = formatter->formatProto(output);
```

Comment:
I wasn't calling it out in other pre-existing locations, but we should try to avoid using auto unless the type is fully obvious from context (which it isn't here).  Can you please declare the full type?